open-distro-elasticsearch-kubernetes/helm/opendistro-es/templates/kibana/kibana-service.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.kibana.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
```

Comment:
Include `annotations` in order to allow users to define extra annotations for external LB.